modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/servlet/taglib/ChangeTrackingIndicatorDynamicInclude.java

```diff
@@ -355,6 +402,27 @@ else if (unsupportedApplication) {
 				data.put("warningLearnLink", null);
 				data.put("warningButton", true);
 			}
+			else if (showContextChangePopover) {
+				data.put(
+					"title",
+					StringBundler.concat(
+						ctCollection.getName(), " (",
+						_language.get(
+							themeDisplay.getLocale(),
+							"keep-working-in-this-publication"),
+						")"));
```

Comment:
The message is not added to the publication bar in the mockup
<img width="904" alt="Screen Shot 2024-08-16 at 9 04 02 AM" src="https://github.com/user-attachments/assets/dd696e57-109c-4656-b175-d32d79a28a10">

Reply:
This is how it looks now
![image](https://github.com/user-attachments/assets/08e527f4-b9fb-4de8-aa83-3ba4f7468452)